### DISCUSSION TITLE
Arabic font in data.yaml

### DISCUSSION THREAD

**Discussion by asadikani** [ultralytics/ultralytics#2093] [Q&A]
how can i use Arabic labels in data.yaml file?

**Comment by glenn-jocher** [comment]
@asadikani hello! To use Arabic labels in your `data.yaml` file, you should ensure that the file is saved with UTF-8 encoding, which supports Arabic characters. Most modern text editors will allow you to choose UTF-8 encoding when saving a file. Once you've confirmed the encoding, you can simply type your Arabic labels directly into the `data.yaml` file where you define your class names. Make sure that your environment and tools are set up to handle UTF-8 encoded text to avoid any issues during training or inference. Good luck with your project! 😊🚀
